styles/styles.css

```diff
@@ -0,0 +1,289 @@
+button:hover {
+    cursor: pointer;
+}
+
+img {
+    width: 100%;
+}
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+/* toen ik hier met kleuren ging gaan mij LI mee omdat die ook paars zijn die straks nog naar paars */
+.header-section{
+    background: var(--color-purple);
+    color: var(--color-light-grey);
+}
+
+.header-navigation{
+    width: 100%;
+    display: flex;
+    justify-content: flex-end;
+    padding: 20px 0;
+}
+
+.header-navigation ul {
+    list-style: none;
+    display: flex;
+    flex-basis: 400px;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.header-navigation ul li a {
+    color: var(--color-light-grey);
+}
+
+.header-content{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    padding: 60px 120px;
+    row-gap: 20px;
+}
+
+/* --------------- 4.2 Introduction ------------------- */
+
+.introduction{
+    /*flex-basis: 600px;*/
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    padding: 60px 120px;
+    /*row-gap: 50px;*/
+}
+
+.introduction p {
+    padding: 60px 0 40px 0;
+}
+
+.introduction h2{
+    color: var(--color-purple);
+    padding: 10px 0;
+}
+
+
+/* --------------- 4.3 Projects ------------------------- */
+
+.projects-section-wrapper{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 40px 0;
+}
+
+.projects-content-text{
+    flex-basis: 400px;
+}
+
+
+.projects-content-image{
+    flex-basis: 800px;
+}
+
+.projects-content-text img {
+    max-width: 50px;
+}
+
+.reverse {
+    flex-direction: row-reverse;
+}
+/* --------------- 4.4 Services-grid ------------------------- */
+.grid-section{
+
+}
+
+.inner-container-grid {
+    flex-basis: 1400px;
+    flex-direction: row;
+    display: flex;
+    margin: 0 20px;
+    padding: 100px 0;
+    row-gap: 20px;
+    justify-content: space-between;
+    flex-wrap: wrap;
+}
+
+.grid-article{
+    flex-direction: row;
+    flex-basis: 400px;
+    border: 1px solid var(--color-purple);
+    padding: 20px 40px;
+
+}
+
+.grid-article h3{
+    color: var(--color-purple);
+}
+
+.grid-section img {
+    max-width: 50px;
+    margin: 16px 0;
+}
+
+/* --------------- 4.5 Contact form  ------------------------- */
+
+.form-section{
+    background: var(--color-purple);
+}
+
+.form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    color: var(--color-light-grey);
+}
+
+.form form {
+    flex-basis: 600px;
+    width: 50%;
+
+    display: flex;
+    flex-direction: column;
+    padding: 40px 0;
+}
+
+form h2 {
+    text-align: center;
+    padding: 20px 0;
+}
+
+form h5 {
+    text-align: center;
+    padding: 20px 0;
+}
+
+form label {
+    padding: 10px 0;
+}
+
+form input{
+    padding: 10px 0;
+}
+form button {
+    width: 20%;
+}
+
```

Comment:
Wanneer je het form element een class had gegeven zou je daar hier de styling voor kunnen opschrijven. 

form{
    display: flex;
    flex-direction: column;
    text-align: left;
    width: 100%;
}

Hiermee zetten we het formulier onder elkaar en alles gecentreerd. Wel willen we de tekst van de input elementen links hebben, vandaar text-align: left;.

Vervolgens kan je nu de button een align-self van center geven zodat die wel gecentreerd blijft en voila. De button staat nu netjes in het midden onderaan ^^